Source/Foundation/NSObject.swift

```diff
@@ -34,4 +34,29 @@ extension NSObject {
             .rac_willDeallocSignal()
             .rex_toTriggerSignal()
     }
+	
+	
+	/// Returns a signal which will send a tuple of arguments upon each invocation of
+	/// the selector, then completes when the receiver is deallocated. `next` events
+	/// will be sent synchronously from the thread that invoked the method. If
+	/// a runtime call fails, the signal will send an error in the
+	/// RACSelectorSignalErrorDomain.
+	/// It can be used for example to catch `@IBAction func`.
+	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
+	public func rex_signalForSelector(selector: Selector) -> Signal<(AnyObject?, AnyObject?, AnyObject?, AnyObject?, AnyObject?, AnyObject?), NSError> {
```

Comment:
Since we're in Swift's lands why not use generics here instead of `AnyObject`? 😊